### DISCUSSION TITLE
cs va ow 等各种游戏 无法打字怎么办？还是说用这输入法的没人玩游戏？

### DISCUSSION THREAD

**Discussion by Eff0rtly** [rime/home#1657] [Q&A]
cs va ow 等各种游戏 无法打字怎么办？还是说用这输入法的没人玩游戏？

**Comment by Yooupi** [comment]
我也有这个问题，在CS2中使用小狼毫会引起闪退bug，使用微软输入法没这个问题；
请问你找到解决办法了吗？

**Reply by Eff0rtly** [reply]
解决了 很简单 就是不用这输入法

**Reply by Yooupi** [reply]
好吧，我现在是平时用小狼毫，游戏使用微软输入
